index.js

```diff
@@ -43,6 +43,9 @@ module.exports = function hsts (options) {
   }
 }
 
+module.exports = hsts;
+module.exports.hsts = hsts;
```

Comment:
What is the rationale for this? If this makes things easier, I could consider including it for all of Helmet's modules.